### PR TITLE
partial improvement for issue #4

### PR DESCRIPTION
**Pull request by macrzen**
Partial improvement for issue #4 
Changes to:
- Track
-- setGridPane & updateStats methods.
-- added** two lists to contain the labels for locations and cars respectively.
-- reference to the active-car label created.

- Car
-- modified the car so its number displayed starts at 1 instead of 0.

### CODE REVIEW

File: src/main/java/Car.java [hunk 1]
```diff
@@ -70,7 +70,7 @@ public void newLocation(double x, double y, double distanceIn) {
      */
     @Override
     public String toString() {
-        return "Car #:" + id + "\tTime traveled: " + time;
+        return (id + 1) + "\t\t" + String.format("%.1f", time);
```
Comment (aaleksanyan) [inline src/main/java/Car.java:73]: What does String.format... do? I see that id+1 makes it so that it's car 1 instead of car 0.

Comment (macrzen) [reply src/main/java/Car.java:73]: String.format is very similar to how printf formats strings.
The big difference is that String.format returns a String whereas printf returns a void.
In this case, %.1f , means to take the double called time, and round it to the 1 decimal place.

Comment (aaleksanyan) [reply src/main/java/Car.java:73]: Oh wow that's actually pretty sick. I didn't know that you could do that in Java.

**Review by aaleksanyan — APPROVED** [review]
Looks good!

**Pull request merged**